packages/paste-core/components/menu/src/index.tsx

```diff
@@ -49,9 +51,12 @@ const StyledMenuItem = React.forwardRef<HTMLDivElement, BoxProps>(({children, ..
   return (
     <Box
       {...safelySpreadBoxProps(props)}
+      as="a"
```

Comment:
Should we make this `as="button"` if an href isn't passed?

Reply:
If an href is passed, it'd be great if we can extract this code from the Anchor component and re-apply it here:
https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/anchor/src/index.tsx#L53

Reply:
> Should we make this as="button" if an href isn't passed?

Honestly, probably more effort than it's worth. You then have to conditionally remove all the browser button styles. Even swapping between a div and an anchor might not necessarily be worth it, considering an anchor does everything we need in both situations. If you don't pass the href the anchor just acts like a div, doesn't even expose the "open in..." context menu options in chrome and firefox. Doesn't seem to have any side effects.

> If an href is passed, it'd be great if we can extract this code from the Anchor component and re-apply it here

For that, I would say that's out of scope of this ticket. I'll make a JIRA to abstract and handle external links in the menu for another time. I'll remove the `target` prop as it should be handled by the abstracted functionality. Plainly linking to a page is fine for the alpha version of this component

Reply:
Abstracted the securing of external links into this PR